### ISSUE TITLE
Date format crashing SMP

### ISSUE THREAD

**Issue by Artins90** [kbuffington/Georgia#99]
I added a new track and every time I play it, SMP crashes with the following message: 
```
Error: Spider Monkey Panel v1.4.1 (Georgia: Georgia v2.0.2 by Mordred)
include failed:

'date' has invalid value: 2015-19-11
in calcAgeDateString()


File: Common.js
Line: 501, Column: 3
Stack trace:
  ArgumentError@Common.js:501:3
  calcAgeDateString@helpers.js:288:10
  on_metadb_changed@georgia-main.js:1630:17
  on_playback_new_track@georgia-main.js:1542:3
  on_init@georgia-main.js:1336:3
  @georgia-main.js:345:1
  loadAsyncFile/</<@georgia-theme.js:9:13
```
Foobar keeps working, I can listen to the song after SMP crashes and displays the "Aw crashed" red color fill.
Deleting the date from the tag solves the issue.

**Comment by kbuffington** [comment]
The issue is that your dates have the month and date transposed for proper handling in JavaScript. Should be "2015-11-19". 

I should probably prevent this from crashing since you're now the 2nd person who has had this issue. Unfortunately there's no good way to localize this properly that I can think of, but I'll give it a look.

**Comment by kbuffington** [comment]
Fixed in 2.0.3.
